game/scripts/npc/heroes/chatterjee.txt

```diff
@@ -63,12 +63,12 @@
     "AttributeBaseAgility"      "13"
     "AttributeAgilityGain"      "2.0"
     "AttributeBaseIntelligence" "25"
-    "AttributeIntelligenceGain" "3.5"
+    "AttributeIntelligenceGain" "3.0"
```

Comment:
I don't think this needs a nerf. Medusa's INT is 23 + 3.4 for reference. Stat growth changes affect Chatterjee's mid and late game.

Reply:
Yeah I'm gonna fiddle around with this a bit more, my new idea is to make a talent for her shield damage absorption so she could potentially scale better with items in the mid game. Gotta learn how to make it though, should be fun